### PR TITLE
Add appropriate tab, tablist, and tabpanel attributes to left drawer

### LINKED ISSUE #1677: Add roles for tab, tablist & tabpanel to the media viewer
The buttons on the left side function as tabs so we should add the appropriate roles there. 

See https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role

The tab should also contain the `aria-controls` attribute and the `aria-selected` attribute

### PR DESCRIPTION
**Pull request by aaron-collier**
Fixes #1677

### CODE REVIEW

File: app/components/embed/media_with_companion_windows_component.html.erb [hunk 1]
```diff
@@ -43,33 +43,33 @@
 
   <div class="media-component-body">
     <aside class="left-drawer collapse" data-media-target="leftDrawer" id="left-drawer">
-      <nav class="vert-tabs">
-        <button data-action="click->media#displayMetadata" data-media-target="leftButton" class="active" aria-label="About this item" aria-controls="about">
+      <nav class="vert-tabs" role="tablist">
+        <button data-action="click->media#displayMetadata" data-media-target="leftButton" class="active" aria-label="About this item" aria-controls="about" aria-selected="true" role="tab">
```
Comment (jcoyne) [inline app/components/embed/media_with_companion_windows_component.html.erb:47]: Can you please add the logic to change the `aria-selected` when one of the other tabs is pressed?

**Review by jcoyne — CHANGES_REQUESTED** [review]

File: app/components/embed/media_with_companion_windows_component.html.erb (hunk 1 above)
Comment (aaron-collier) [reply app/components/embed/media_with_companion_windows_component.html.erb:47]: Done.

File: app/javascript/packs/media.js [hunk 2]
```diff
@@ -15,4 +15,25 @@ $(document).ready(function() {
...
+
+  // Add a click event handler to each tab
+  tabs.forEach((tab) => {
+    tab.addEventListener("click", changeTabs);
+  });
 });
+
+function changeTabs(e) {
+  const target = e.target;
+  const parentButton = target.parentNode;
```
Comment (jcoyne) [inline app/javascript/packs/media.js:29]: This seems a bit fragile as it's tightly bound to the structure of the dom.  Can you use `.closest('[role="tabitem"]')` here and same for `tablist` below?

Comment (aaron-collier) [reply app/javascript/packs/media.js:29]: Sure - I think this makes sense for the tablist select, I'm curius about this for parent... the issue is that the click event happens on the svg object, not the button, so I need to grab the parent button of the svg item... would `closest` be sufficient? And is `tabitem` correct vs `tab`?

I'll work through it and see what happens.

Comment (aaron-collier) [reply app/javascript/packs/media.js:29]: Figured it out, I think it does need to be `tab` :-)

**Review by jcoyne — APPROVED** [review]

**Pull request merged**